python.py

```diff
@@ -0,0 +1,3 @@
+(self, o):
```

Comment:
IndentationError: unindent does not match any outer indentation level<br>SyntaxError: invalid syntax